design/sd-yaml.md

```diff
@@ -41,7 +41,7 @@ Example:
 | `blockedBy` |  job config | syntax will be something like `blockedBy: [~sd@123:main]` (similar to `requires`) so it should live in the same place |
 | `buildPeriodically` | annotation | only works with executor queue plugin |
 | `freezeWindow` | annotation | similar to `buildPeriodically` |
-| `repoManifest` | annotation | only works with Github plugin |
+| `repoManifest` | annotation | only works with GitHub plugin |
```

Comment:
I'm very guilty of writing `GitHub` as `Github`
Thanks for fixing it here
not sure if it's worth fixing it everywhere else as well though

Reply:
I'm happy to fix it elsewhere if you'd like.